sdk/script/account-manager.ts

```diff
@@ -9,12 +8,12 @@ var packageJson = require("../../package.json");
 
 declare var fs: any;
 
-if (typeof window === "undefined") {
-    fs = require("fs");
+if (typeof window === `undefined`) {
```

Comment:
I don't think we should use template strings for everything.  As with other language features, we should use template strings when their use is appropriate and their benefits are realized.